src/metrics/metric_test.go

```diff
@@ -15,6 +17,17 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMain(m *testing.M) {
+	// TODO remove global args.
+	// This test are heavily based on global args to create entities.
+	args.GlobalArgs = args.RabbitMQArguments{
```

Comment:
I am not getting what we are testing here 🤔

Reply:
that was added to the cluster_test.go to test that the entity names contains the host and the port , but also affects other tests where that is being checked since it affects all entities. 

Since I'm removing the `cluster_test.go` file and the TestMain applies to all package tests I needed to add it again.

Reply:
ah ok I did not notice it was a test main 😶‍🌫️